tests/testautonatv2.nim

```diff
@@ -107,3 +119,32 @@ suite "AutonatV2":
 
     # DialBackResponse
     checkEncodeDecode(DialBackResponse(status: DialBackStatus.Ok))
+
+  asyncTest "Successful DialRequest":
+    let
+      src = newStandardSwitch()
```

Comment:
not really sure if we should use `newStandardSwitch` here (because of comment from above).  
- changes in newStandardSwitch can easily break this code
- sure there is comment; but we should avoid it if possible 

could we have `newStandardSwitchBulder()`; that is called in `newStandardSwitch` and `createAutonatV2Switch` should use it and just add autonav2??